tests/dualtor/test_tor_ecn.py

```diff
@@ -213,19 +232,20 @@ def verify_ecn_on_received_packet(
         logging.info("the expected ECN: {0:02b} matching with received ECN: {0:02b}".format(exp_ecn, rec_ecn))
 
 def test_dscp_to_queue_during_decap_on_active(
-    apply_active_state_to_orchagent,
-    build_encapsulated_ip_packet,
-    rand_selected_interface, 
-    ptfadapter,
-    tbinfo, 
-    rand_selected_dut, 
-    tunnel_traffic_monitor, 
-    duthosts, 
-    rand_one_dut_hostname
+    setup_dualtor_tor_active,
+    build_encapsulated_ip_packet, request,
+    rand_selected_interface, ptfadapter,
+    tbinfo, rand_selected_dut, tunnel_traffic_monitor, 
+    duthosts, rand_one_dut_hostname
 ):
     """
     Test if DSCP to Q mapping for inner header is matching with outer header during decap on active
     """
+    if is_t0_mocked_dualtor(tbinfo):
+        request.getfixturevalue('apply_active_state_to_orchagent')
+    else:
+        request.getfixturevalue('toggle_all_simulator_ports_to_rand_selected_tor')
+
```

Comment:
These lines  should be redundant now that you have `setup_dualtor_tor_active`

Reply:
removed.